client/question/services/FeedbackGeneratorServiceSpec.js

```diff
@@ -1279,5 +1345,35 @@ describe('FeedbackGeneratorService', function() {
       }
     );
   });
-});
 
+  describe('_getRandomInt', function() {
+    it('should return a random number within the defined range',
+      function() {
+        var min = 0;
+        var max = 4;
+        var maxTries = 100;
+        for (var i = 0; i < maxTries; i++) {
+          var randomNumber = FeedbackGeneratorService._getRandomInt(min, max);
+          expect(typeof randomNumber).toEqual('number');
+          expect(randomNumber).toBeGreaterThan(-1);
+          expect(randomNumber).toBeLessThan(4);
+        }
+      }
+    );
+  });
+
+  describe('_getCorrectnessFeedbackString', function() {
+    it('should return a feedback string from CORRECTNESS_FEEDBACK_TEXT',
+      function() {
+        var selectedFeedbckText = '';
```

Comment:
I don't think you need this declaration; just put a var two lines below.

Reply:
Fixed